go/memcached-operator/pkg/controller/memcached/memcached_controller.go

```diff
@@ -218,34 +197,11 @@ func (r *ReconcileMemcached) deploymentForMemcached(m *cachev1alpha1.Memcached)
 			},
 		},
 	}
-	// Set Memcached instance as the owner of the Deployment.
+	// Set Memcached instance as the owner and controller
```

Comment:
Revert this comment change. I think it is correct as is.

Reply:
Reverted.